docs/epic/V3.0-COMPLETION-PLAN.md

```
@@ -375,6 +405,40 @@ DELETE  .opencode/USER/README.md   (nach dev-Reorganisation entfernt)
 ---
 
+### PR-12: Semantische Claude→OpenCode Bereinigung (NEU)
+**Branch:** `release/v3.0-pr12-claude-semantic-cleanup`
+**Dateien:** 16 (alle MODIFY — bereits auf main, identisch mit dev)
+**CodeRabbit-Fokus:** Semantische Korrektheit der OpenCode-Beschreibungen, kein "Claude Code" als Plattformname
+
+> **Detailplan:** `docs/epic/CLAUDE-CLEANUP-PLAN.md`
+
+**Warum eigener PR:** Die schwersten Claude-Dateien (THEHOOKSYSTEM.md 48 Treffer, TOOLS.md 25, MEMORYSYSTEM.md 20, SKILLSYSTEM.md 17) sind bereits identisch auf main und dev. Sie sind in keinem der 11 PRs enthalten.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)